possel/configuration.py

```diff
@@ -0,0 +1,30 @@
+import yaml
+
+
+class Configuration:
+    def __init__(self, default={}):
```

Comment:
don't pass a mutable value as a default argument, this will cause all instances to share the same dict, see http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments !fix